variants/frontend-typescript/types.d.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/order */
```

Comment:
🐘 at some point I'll PR these rail modules to DefinitelyTyped, but for now this'll do